perf-tests/clusterloader2/pkg/framework/client/objects.go

```diff
@@ -147,10 +147,10 @@ func RetryFunction(f func() error, options ...*ApiCallOptions) wait.ConditionFun
 }
 
 // ListPodsWithOptions lists the pods using the provided options.
-func ListPodsWithOptions(c clientset.Interface, namespace string, listOpts metav1.ListOptions) ([]apiv1.Pod, error) {
+func ListPodsWithOptions(c clientset.Interface, tenant, namespace string, listOpts metav1.ListOptions) ([]apiv1.Pod, error) {
```

Comment:
Same question here and elsewhere applicable - we should probably test it for scale-up scenario - deploy a regular cluster and run perf test as we do in the scale-up case.

Reply:
This code change has been verified in scale-up scenario.